internal/store/validatingwebhookconfiguration.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package store
+
+import (
+	admissionregistration "k8s.io/api/admissionregistration/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/kube-state-metrics/pkg/metric"
+)
+
+var (
+	descValidatingWebhookConfigurationName          = "kube_validatingwebhookconfiguration_labels"
+	descValidatingWebhookConfigurationHelp          = "Kubernetes labels converted to Prometheus labels."
+	descValidatingWebhookConfigurationDefaultLabels = []string{"namespace", "validatingwebhookconfiguration"}
+
+	validatingwebhookconfigurationFamilies = []metric.FamilyGenerator{
+		{
+			Name: "kube_validatingwebhookconfiguration_info",
+			Type: metric.Gauge,
+			Help: "Information about the ValidatingWebhookConfiguration.",
+			GenerateFunc: wrapVWCFunc(func(vwc *admissionregistration.ValidatingWebhookConfiguration) *metric.Family {
+				return &metric.Family{
+					Metrics: []*metric.Metric{{
+						LabelKeys:   []string{},
+						LabelValues: []string{},
+						Value:       1,
+					}},
+				}
+			}),
+		},
+		{
+			Name: "kube_validatingwebhookconfiguration_created",
+			Type: metric.Gauge,
+			Help: "Unix creation timestamp.",
+			GenerateFunc: wrapVWCFunc(func(vwc *admissionregistration.ValidatingWebhookConfiguration) *metric.Family {
+				ms := []*metric.Metric{}
+
+				if !vwc.CreationTimestamp.IsZero() {
+					ms = append(ms, &metric.Metric{
+						LabelKeys:   []string{},
+						LabelValues: []string{},
+						Value:       float64(vwc.CreationTimestamp.Unix()),
+					})
+				}
+				return &metric.Family{
+					Metrics: ms,
+				}
+			}),
+		},
+		{
+			Name: "kube_validatingwebhookconfiguration_metadata_resource_version",
+			Type: metric.Gauge,
+			Help: "Resource version representing a specific version of the ValidatingWebhookConfiguration.",
```

Comment:
```suggestion
			Help: "Resource version representing a specific version of the validating webhook configuration.",
```

Reply:
Why space separate it? The official resource name is `MutatingWebhookConfiguration` in the k8s API. 

Also, keeping it as one word will keep it consistent with other resource representations in this repo. 
[example](https://github.com/kubernetes/kube-state-metrics/blob/master/internal/store/persistentvolume.go#L102)

Reply:
Looks like we haven't been too consistent with the naming. There are other examples where we space-separate it, for e.g:- limit ranges, resource quota, persistent volume claims. So either as-is/space-separating would work.